Popular_Models_Guide/StableDiffusion/README.md

```diff
@@ -57,21 +57,21 @@ support matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/i
 ## Building the Triton Inference Server Image
 
 The example is designed based on the
-`nvcr.io/nvidia/tritonserver:24.01-py3` docker image and [TensorRT OSS v9.2.0](https://github.com/NVIDIA/TensorRT/releases/tag/v9.2.0).
+`nvcr.io/nvidia/tritonserver:YY.MM-py3` docker image and [TensorRT OSS v9.2.0](https://github.com/NVIDIA/TensorRT/releases/tag/v9.2.0).
 
 A set of convenience scripts are provided to create a docker image
-based on the `nvcr.io/nvidia/tritonserver:24.01-py3` image with the
+based on the `nvcr.io/nvidia/tritonserver:YY.MM-py3` image with the
 dependencies for the TensorRT Stable Diffusion demo installed.
 
 ### Triton Inference Server + TensorRT OSS
 
 #### Clone Repository
 ```bash
-git clone https://github.com/triton-inference-server/tutorials.git -b r24.02 --single-branch
+git clone https://github.com/triton-inference-server/tutorials.git --single-branch
```

Comment:
Didn't you want to lock the branch here for reproducibility in blog?

Reply:
I'll add the -b in the blog instructions - here we can leave it as main so it would work as we update it.